### PR TITLE
Fix `ex_docs` syntax

### PR DESCRIPTION
**Pull request by davorbadrov**
This PR fixes some of the documentation syntax to properly display the Example as an `<h2>` tag and it uses multiline `@docs` in some of the payload related examples.

**Before**
error_1
**After**
fixed_1

### CODE REVIEW

**Comment by remi** [comment]
@davorbadrov Thank you for this! Before I merge this, could you run `make format` on the code to make the Travis build pass?

File: lib/absinthe_error_payload/changeset_parser.ex [hunk 1]
```diff
@@ -63,25 +64,25 @@ defmodule AbsintheErrorPayload.ChangesetParser do
     construct_message(field, {message, opts})
   end
 
-  defp construct_field(parent_field, field, options \\ []) do
+  def construct_field(parent_field, field, options \\ []) do
```
Comment (remi) [inline lib/absinthe_error_payload/changeset_parser.ex:67]: Is this change related to the docs?

Comment (davorbadrov) [reply lib/absinthe_error_payload/changeset_parser.ex]: I think this is a typo, I probably changed it by accident - reverted this change.

**Comment by davorbadrov** [comment]
> @davorbadrov Thank you for this! Before I merge this, could you run `make format` on the code to make the Travis build pass?

Sure thing, I've run make format and submitted the changes. Also the defp > def change was reverted.

**Comment by remi** [comment]
Yeah, I think we’ll have to revert to simple strings in those cases. Credo doesn’t seem to like nested `@doc` statements indeed 😉

**Comment by davorbadrov** [comment]
It seems I found a workaround by using sigils. This works both for the syntax, the docs and credo:
```elixir
@doc ~S'''
Some comment

    ```elixir
    @doc """
    Some nested comment for the example
    """
    def some_method() do
      # ...
    end
    ```
'''
```

**Comment by remi** [comment]
Awesome, merging! 👍
